defs/bitcoin/qtum.json

```diff
@@ -0,0 +1,41 @@
+{
+    "coin_name": "Qtum",
+    "coin_shortcut": "QTUM",
+    "coin_label": "Qtum",
+    "website": "https://qtum.org",
+    "github": "https://github.com/qtumproject/qtum/",
```

Comment:
please strip the trailing `/`, here and in testnet

Reply:
fixed